src/components/modules/epics/ContributionsEpic.tsx

```diff
@@ -197,7 +197,8 @@ const EpicBody: React.FC<BodyProps> = ({
 
 export const ContributionsEpicComponent: (
     acVariant: EpicSeparateArticleCountTestVariants,
-) => React.FC<EpicProps> = acVariant => ({
+) => // eslint-disable-next-line react/display-name
+React.FC<EpicProps> = acVariant => ({
```

Comment:
Do we need this ignore? It doesn't seem to be complaining locally for me

Reply:
I suspect this is because a version has changed in the yarn.lock?
Personally I'd prefer adding `"react/display-name": "off"` to the .eslintrc.js. What do you both think?

Reply:
I've reverted the lockfile ✅